test/e2e/query_publisher_sequence.test.ts

```diff
@@ -0,0 +1,40 @@
+import { expect } from "chai"
+import { randomUUID } from "crypto"
+import { connect } from "../../src"
+import { Rabbit } from "../support/rabbit"
+
+describe("query publisher sequence", () => {
+  const rabbit = new Rabbit()
+  const testStreamName = "test-stream"
+  const publisherRef = randomUUID()
+
+  beforeEach(async () => {
+    await rabbit.createStream(testStreamName)
+  })
+
+  afterEach(async () => {
+    await rabbit.deleteStream(testStreamName)
+  })
+
+  it("asking for the last sequence read from a publisher returns the last sequence id", async () => {
```

Comment:
Could you add a test about asking the sequence when nothing has been published?

Reply:
added the test